glide.lock

```diff
@@ -223,7 +223,7 @@ imports:
   - transport
 - name: gopkg.in/src-d/go-errors.v0
   version: 0c303ec4c027302259ec1738f19f515aad25f13f
-- name: gopkg.in/src-d/simple-linguist.v1
+- name: gopkg.in/src-d/enry.v1
```

Comment:
We have to use the sale import that the library defines. Enry does use gopkg.in in it's importa, so we do it too. Glide cannot change that.